Add enablement material for server-side Beta

malomarrec opened the pull request:
Subsumed into https://github.com/sourcegraph/handbook/pull/3936

malomarrec committed:
add enablement material

malomarrec committed:
squash! Prettier

malomarrec committed:
squash! Prettier

malomarrec committed:
link to release post

malomarrec committed:
Merge branch 'mm/enablement-ssbc-beta' of github.com:sourcegraph/handbook into mm/enablement-ssbc-beta

malomarrec committed:
Document platform positioning FAQ (#3937)

cherryhuang7 committed:
how to get involved (#3938)

* how to get involved

* squash! Prettier

Co-authored-by: cherryhuang7 <cherryhuang7@users.noreply.github.com>

thiswayman committed:
Added new team expectations page and updated main product education page. (#3939)

* Updated intro, team expectations,  removed process

Created summary and added a new page for team expectations, removed process items from main page.

* Update contacts area and minor formatting.

* changed page name

* squash! Prettier

Co-authored-by: jwayman <jwayman@users.noreply.github.com>

kemperhamilton committed:
Adjusting dates for exec Q&As (#3940)

Commit:
Update license_keys.md (#3941)

Added a link to the docs on how/where to add the enterprise license key

sanderginn committed:
Adds new MI to template (#3942)

Joelkw committed:
proposal: update standard mileage rate? (#3930)

Standard mileage rate from https://www.irs.gov/newsroom/irs-increases-mileage-rate-for-remainder-of-2022

mpimenov committed:
org chart: add Executive Business Partners (#3943)

milespatton committed:
milespatton-update-zoom.md (#3915)

malomarrec committed:
Move images to bucket

malomarrec committed:
Move images to bucket

malomarrec committed:
remove images

malomarrec committed:
remove images

malomarrec committed:
Merge branch 'main' into mm/enablement-ssbc-beta

malomarrec committed:
Merge branch 'mm/enablement-ssbc-beta' of github.com:sourcegraph/handbook into mm/enablement-ssbc-beta

malomarrec committed:
squash! Prettier

malomarrec committed:
squash! Prettier

malomarrec committed:
SVGs are metter anyways, man

malomarrec committed:
SVGs are metter anyways, man

malomarrec committed:
Merge branch 'mm/enablement-ssbc-beta' of github.com:sourcegraph/handbook into mm/enablement-ssbc-beta

malomarrec committed:
Document value realisation QBR template (#3914)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

collapsableumbrellagirl committed:
Updated how sourcers and recruiters work together (#3946)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

sanderginn committed:
Add executor token with `mg` (#3945)

* Add executor token with `mg`

* Update command

* Add executor check info

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

dnostrand committed:
Updated the WIPs (#3949)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

michaellzc committed:
Update v1.1 executors playbook (#3950)

Co-authored-by: michaellzc <michaellzc@users.noreply.github.com>

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

gabtorre committed:
Update support-schedule.md (#3952)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

jasonhawkharris committed:
Adds engineering-help.md (#3951)

* Create how-to-tickets.md

* squash! Prettier

* Update how-to-tickets.md

* Update how-to-tickets.md

* Update how-to-tickets.md

* squash! Prettier

* Update how-to-tickets.md

* Update content/departments/ce-support/support/process/how-to-tickets.md

Co-authored-by: Quinn Slack <sqs@sourcegraph.com>

* Moves note about How To RFH's to footnote

* squash! Prettier

Co-authored-by: jasonhawkharris <jasonhawkharris@users.noreply.github.com>
Co-authored-by: Quinn Slack <sqs@sourcegraph.com>

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

felixfbecker committed:
Make sure GitHub token is passed around (#3953)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

jasonhawkharris committed:
Adds link to how-to procedure (#3954)

* Update index.md

* Update index.md

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

jasonhawkharris committed:
Update engaging-other-teams.md (#3956)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

emchap committed:
Adding link to 101 (#3957)

* Adding link to 101

* Adding links to 101 recording

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

felixfbecker commented on content/departments/sales/sales-enablement/running-batch-changes-server-side-beta/index.md:
```diff
@@ -0,0 +1,57 @@
+
+A key step in this workflow today is that users run their batch change locally thanks to a local command line interface (`src-cli`). For large amounts of repositories or resource-intensive code rewrites, running `src-cli` locally could take an impractical amount of time and be brittle.
+
+![](local-run.svg)
```
Would recommend using `<object>` over `<img>` for diagrams for accessibility (lets screen readers navigate through the text boxes in the diagram)
```suggestion
<object data="local-run.svg"></object>
```

felixfbecker commented on content/departments/sales/sales-enablement/running-batch-changes-server-side-beta/index.md:
```diff
@@ -0,0 +1,57 @@
+
+[Running batch changes server-side (beta)](https://docs.sourcegraph.com/batch_changes/explanations/server_side) removes the need for running that CLI locally to create the batch changes. Instead, the batch change will be ran on one or multiple separate compute instance.
+
+![](server-side-run.svg)
```
```suggestion
<object data="server-side-run.svg"></object>
```

kalanchan committed:
update: managed instance upgrade list (#3955)

* Update upgrade_managed_issue_template.md

* squash! Prettier

Co-authored-by: kalanchan <kalanchan@users.noreply.github.com>

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

nickycomber committed:
updated Cloud MI OF name (#3961)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

daxmc99 committed:
Add steps to use wireshark with MI (#3962)

Co-authored-by: daxmc99 <daxmc99@users.noreply.github.com>

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

Joelkw committed:
update research template link (#3963)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

Commit:
Update design peoples (#3964)

Adds Paulo to search, removes Marisa, and adds Steve as head and interim admin manager.

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

Commit:
update design peoples (#3965)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

Joelkw committed:
added insights talking points (#3967)

sourcegraph-bot committed:
Merge branch 'main' into mm/enablement-ssbc-beta

malomarrec committed:
Merge branch 'mm/enablement-ssbc-beta' of github.com:sourcegraph/handbook into mm/enablement-ssbc-beta

malomarrec committed:
Update content/departments/sales/sales-enablement/running-batch-changes-server-side-beta/index.md

Co-authored-by: Felix Becker <felix.b@outlook.com>

malomarrec committed:
Merge branch 'main' into mm/enablement-ssbc-beta

malomarrec committed:
add recording

malomarrec committed:
Update content/departments/sales/sales-enablement/running-batch-changes-server-side-beta/index.md

Co-authored-by: Felix Becker <felix.b@outlook.com>

malomarrec commented:
Subsumed into https://github.com/sourcegraph/handbook/pull/3936
